KAKAOWEBTOON-iOS/KAKAOWEBTOON-iOS/Presentation/Home/Cell/AllToonsSectionCell.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  AllToonsSectionCell.swift
+//  KAKAOWEBTOON-iOS
+//
+//  Created by MaengKim on 11/20/24.
+//
+
+import UIKit
+
+import SnapKit
+
+class AllToonsSectionCell: UICollectionViewCell {
+    
+    static let identifier = "AllToonsSectionCell"
+    
+//    private let titleLabel = UILabel(frame: .zero)
+    private let imageView: UIImageView = {
+        let imageView = UIImageView(frame: .zero)
+        imageView.contentMode = .scaleAspectFill
+        imageView.clipsToBounds = true
+        return imageView
+    }()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.contentView.backgroundColor = .gray
+        
+        setUI()
+        setLayout()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        self.contentView.backgroundColor = .gray
+    }
+    
+    private func setUI() {
```

Comment:
노션의 코딩컨벤션 지켜주시면 좋을 것 같습니다.
뷰의 계층구조를 나타내는 함수는 setupHierarchy()로 이름 변경 부탁드립니다.
```suggestion
    private func setupHierarchy() {
```

Reply:
넵 감사합니다